README.md

```diff
@@ -41,6 +41,36 @@ Please refer to [CONTRIBUTING][] before opening an issue or pull request.
 
 See [example_test.go](https://github.com/fsnotify/fsnotify/blob/master/example_test.go).
 
+## FAQ
+
+**When a file is moved to another directory is it still being watched?**
+
+No (it shouldn't be, unless you are watching where it was moved to).
+
+**When I watch a directory, are all subdirectories watched as well?**
+
+No, you must add watches for any directory you want to watch (a recursive watcher is in the works [#56][]).
```

Comment:
we should change this to point to https://github.com/fsnotify/fsnotify/issues/18

(sadly it's not in the works yet)